pom.xml

```diff
@@ -55,6 +55,16 @@
       <artifactId>commons-lang3</artifactId>
       <version>3.5</version>
     </dependency>
+    <dependency>
+        <groupId>commons-codec</groupId>
+        <artifactId>commons-codec</artifactId>
+        <version>1.10</version>
+    </dependency>
+    <dependency>
+        <groupId>javax.xml.ws</groupId>
+        <artifactId>jaxws-api</artifactId>
+        <version>2.3.1</version>
+    </dependency>
```

Comment:
Why do we need these dependencies?

Reply:
Maven said those dependencies were missing once I began compiling for java 11.

I'll try removing them now that there are no other errors to make sure they're necessary.

Reply:
The build still fails with compilation errors when those dependencies aren't explicitly added under java 11.

Reply:
Looks like javax.xml.ws was deprecated in 9 and removed in 11:
https://jaxenter.com/jdk-11-java-ee-modules-140674.html

Reply:
Make sense, thanks!